ibm_catalog.json

```diff
@@ -1426,13 +1442,49 @@
               "key": "workers_per_zone"
             },
             {
-              "key": "wait_till"
+              "key": "wait_till",
+              "type": "string",
+              "type_metadata": "string",
+              "default_value": "IngressReady",
```

Comment:
Don't specify a default here - allow it to inherit the default value in terraform